test/test_sparse.py

```diff
@@ -246,6 +246,17 @@ def test_sparse_sum():
         ref = test_sparse_sum()
         self.assertTrue(ref.expired())
 
+    @onlyCPU
+    @dtypes(torch.double)
```

Comment:
`torch.float` for consistency

Reply:
Notice that test_sparse.py uses mostly `torch.double` making the usage of `torch.float` less consistent.

If `torch.float` consistency is still desired, it should be applied to the whole test file. This would be OT for this PR and should be addressed in another PR, IMHO.

Reply:
OK